modules/setting-up-cpu-manager.adoc

```diff
@@ -96,9 +96,10 @@ sh-4.2# cat /host/etc/kubernetes/kubelet.conf | grep cpuManager
 [source,terminal]
 ----
 cpuManagerPolicy: static        <1>
-cpuManagerReconcilePeriod: 5s   <1>
+cpuManagerReconcilePeriod: 5s   <2>
 ----
-<1> These settings were defined when you created the `KubeletConfig` CR.
+<1> This setting is defined when you create the `KubeletConfig` CR.
```

Comment:
```suggestion
<1> cpuManagerPolicy is defined when you create the `KubeletConfig` CR.
```
You can add field names for both since they are in diff callouts.

Reply:
Addressed